MacrobenchmarkSample/gradle.properties

```diff
@@ -33,3 +33,4 @@ org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
 android.useAndroidX=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+android.enableAdditionalTestOutput=true
```

Comment:
this should be the default for at least arctic fox - can you check if this is needed?

Reply:
Hmm, it behaves weird, If I run it with `./gradlew :benchmark:cC`
It copies all the .perfetto-trace files (not the benchmarkData.json files), but when it runs `benchmarkReport` task it clears the folder and won't copy anything so there are no outputs.

Reply:
If I run it for macrobenchmark, then when running from AS, it doesn't copy anything (with or without the flag), but running from gradle, it copies properly (without the flag)